app/resources/admin_api_key_form.html.template

```diff
@@ -204,10 +207,20 @@
 
   <p>
     <input type="hidden" name="key" value="{{target_key.key}}">
-    <input type="submit" name="submit" value="{{operation_name}}" id="submit"
-           onclick="return validate_fields();">
-    <div id="mandatory_field_missing"
-         class="mandatory" style="display: none">
+    {% if operation_type == 'update' %}
+      <input
+          type="submit"
```

Comment:
nit: this indentation is different than the following lines?

Reply:
Fixed (it was the other lines that were wrong).